ci/scenarios/vPool/add_remove_vpool_concurrently/main.py

```diff
@@ -72,13 +71,13 @@ def add_vpools(self):
                                                    'vpool_name': vpool_name,
                                                    'writecache_size': 5}})
             tasks[vpool_name] = self.api.post(api='storagerouters/{0}/add_vpool'.format(sr.guid), data=data)
-        for vpool_name, task in tasks.iteritems():
-            self.LOGGER.info('-> {0} running: {1}'.format(vpool_name, self.api.wait_for_task(task)[0]))
+        for vpool_name, task_id in tasks.iteritems():
+            self.LOGGER.info('-> {0} running: {1}'.format(vpool_name, self.api.wait_for_task(task_id, timeout=600)[0]))
```

Comment:
You should do the same here as in `remove_vpools` and verify the result of the `wait_for_task`